### DISCUSSION TITLE
VendorOrders functions for vendors

### DISCUSSION THREAD

**Discussion by dave-sewell** [abuzuhri/Amazon-SP-API-CSharp#313] [Q&A]
Is there an estimated timeline for adding VendorOrders functions for vendors to the library?

If there are not plans to add those in the next few months, we are considering developing and contributing code to add those.

This library has been very helpful to us. Thanks.

**Comment by abuzuhri** [comment]
> Is there an estimated timeline for adding VendorOrders functions for vendors to the library?
> 
> If there are not plans to add those in the next few months, we are considering developing and contributing code to add those.
> 
> This library has been very helpful to us. Thanks.

@dave-sewell  thanks to use my library 

I try to build this library when I have free time and its getting bigger that I expect and I need help if you can add it and I will merge your code to the library
